app/controllers/hq/orphans_controller.rb

```diff
@@ -23,6 +24,18 @@ def update
 
 private
 
+  def load_scope
+    @orphans_all = Orphan.count
+    @orphans_sort_by_eligibility = Orphan.sort_by_eligibility.count
```

Comment:
Names of vars are confusing.
Maybe `orphans_count`  and `orphans_sort_by_eligibility_count`?